txpool/txpool.go

```diff
@@ -189,6 +199,7 @@ func NewTxPool(
 		gauge:       slotGauge{height: 0, max: config.MaxSlots},
 		priceLimit:  config.PriceLimit,
 		sealing:     config.Sealing,
+		whitelists:  whitelists,
```

Comment:
If whitelists won't change dynamically, you can use map instead of slice for faster lookup.

Reply:
Fixed 6974e01cbb5bd4d4c82039d9a5decc59aba85c87